package.json

```diff
@@ -30,8 +30,8 @@
     "repl": "node tools/repl.js",
     "release": "ts-node-script scripts/release/cli.ts",
     "pretest": "node tools/pretest.js",
-    "test": "lerna run --ignore firebase-repo-scripts-prune-dts --ignore firebase-messaging-integration-test --concurrency 4 --stream test",
-    "test:ci": "lerna run --concurrency 4 test:ci",
+    "test": "lerna run --ignore firebase-messaging-integration-test --concurrency 4 --stream test",
```

Comment:
The firebase-repo-scripts-prune-dts is ok to run now?

Reply:
Ran it locally without an issue.